tests/utils/tab.spec.js

```diff
@@ -28,6 +29,7 @@ function addTabs() {
 describe('tabUtils', () => {
 	beforeEach(() => {
 		tabUtils.tabs = [];
+		tabUtils.config = {};
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2 tabs.](https://app.codacy.com/gh/wingify/across-tabs/pullRequest?prid=7659180)
```suggestion
    tabUtils.config = {};
```